docs/api-versioning.md

```diff
@@ -10,13 +10,14 @@ The version of the REST API should be explicitly specified in the request URL as
 
 `https://<service_url>/v<version>/studies`
 
-Currently routes without a version are still supported (ex. `https://<service_url>/studies`) and has the same behavior as specifying the version as v1.0-prerelease. However, we strongly recommended to specify the version in all requests via the URL.
+Currently routes without a version are still supported (ex. `https://<service_url>/studies`) and has the same behavior as specifying the version as v1. However, we strongly recommended to specify the version in all requests via the URL as routes without a version will not be supported after General Availability.
 
 
 ### Supported Versions
 
 Currently the supported versions are:
-- v1.0-prerelease
+- v1-prerelease
```

Comment:
I don't think we can change this version since we already published it - should stay as `v1.0-prerelease`

Reply:
Reverted this change.